src/util/selinux.c

```diff
@@ -66,14 +65,12 @@ int selinux_file_context(const char *dst_name)
             ret = errno;
             DEBUG(SSSDBG_CRIT_FAILURE, "Unable to lookup selinux context "
                   "[%d]: %s", ret, sss_strerror(ret));
-            ret = 1;
             goto done;
         }
 
         /* Set the security context for the next created file */
         if (setfscreatecon(scontext) < 0) {
             if (security_getenforce() != 0) {
-                ret = 1;
```

Comment:
Hi,

if I understand the flow correctly `ret`can be `0` here if `selabel_lookup()` was successful, so it might be more reliable to set `ret` here explicitly.

bye,
Sumit